.pre-commit-config.yaml

```
@@ -37,3 +37,6 @@ repos:
     hooks:
       - id: poetry-check
         args: ['-C', 'api']
+
+ci:
+  autoupdate_commit_msg: 'ci: pre-commit autoupdate'
```

No need to include this in the changelog, IMO:

```suggestion
  autoupdate_commit_msg: 'chore(ci): pre-commit autoupdate'
```

---

Happy to change this if we want, but `ci` changes _aren't_ included in the changelog.

---

See [here](https://github.com/Flagsmith/flagsmith/blob/main/release-please-config.json#L33).